### PR TITLE
ft(coveralls-integration): add coveralls

### PR DESCRIPTION
**Pull request by izabayogit**
#### What does this PR do?
Integrate coveralls 
#### Description of Task to be completed?
- link coveralls with github repo
- integrate coveralls in codes
#### How should this be manually tested?
- clone git repo
-  npm run test
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
- #170766077
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Comment by KKDoddy** [comment]
Looks good to me

**Comment by karengir** [comment]
good work! I think there's no issue.

**Review by karengir — DISMISSED** [review]
that's good, nothing to add.

**Comment by jajajaden01** [comment]
Looks good on my side.

**Review by jajajaden01 — DISMISSED** [review]
Good job on my side

**Review by KKDoddy — DISMISSED** [review]

**Review by Nigorjeanluc — DISMISSED** [review]
It looks perfect to me

**Review by Nigorjeanluc — DISMISSED** [review]

**Review by karengir — DISMISSED** [review]

**Comment by key-joshua** [comment]
Jonas should you correct your PR's description because there are some mistakes, Check well again !!

jonas

The same-time your PR have conflicts which are needed to be resolved

jonas conflict

**Review by jajajaden01 — DISMISSED** [review]

**Review by jajajaden01 — DISMISSED** [review]

**Review by Nigorjeanluc — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,24 +1,77 @@
 {
   "name": "ateam-bn-backend",
   "version": "1.0.0",
-  "description": "We make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.",
```
Comment (YankyJayChris) [inline package.json]: resolve this conflict

File: package.json [hunk 2]
```diff
@@ -1,24 +1,77 @@
...
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage",
+    "test:coveralls": "jest --coverage --coverageReporters=text-lcov | coveralls"
+<<<<<<< HEAD
+<<<<<<< HEAD
+>>>>>>> ft(coveralls-integration): add coveralls
+=======
+>>>>>>> ft(coveralls-integration): add coveralls
+=======
+>>>>>>> ft(coveralls-integration): add coveralls
```
Comment (YankyJayChris) [inline package.json]: resolve this conflict

File: .travis.yml [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+>>>>>>> ft(coveralls-integration): add coveralls
+=======
+>>>>>>> ft(coveralls-integration): add coveralls
+sudo: false
 language: node_js
 node_js:
-  - 'stable'
-script:
-  - 'npm run test'
+  - node
```
Comment (YankyJayChris) [inline .travis.yml]: work on this conflict

**Review by YankyJayChris — CHANGES_REQUESTED** [review]

**Review by Nigorjeanluc — DISMISSED** [review]

**Review by KKDoddy — DISMISSED** [review]

**Review by jajajaden01 — DISMISSED** [review]

**Review by karengir — DISMISSED** [review]

**Review by KKDoddy — DISMISSED** [review]

**Review by Nigorjeanluc — DISMISSED** [review]

**Review by YankyJayChris — DISMISSED** [review]

**Review by jajajaden01 — DISMISSED** [review]

**Review by karengir — DISMISSED** [review]

**Review by key-joshua — DISMISSED** [review]

File: _BACKUP_83.gitignore [hunk 4]
```diff
@@ -0,0 +1,6 @@
+node_modules
+<<<<<<< HEAD
+=======
```
Comment (luc-tuyishime) [inline _BACKUP_83.gitignore:3]: Why these conflicts codes ??

File: .travis.yml [hunk 5]
```diff
@@ -1,5 +1,28 @@
...
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (luc-tuyishime) [inline .travis.yml]: Resolve the conflicts properly.

**Review by Nigorjeanluc — DISMISSED** [review]

**Pull request merged**